master/usage/assigning-ip-addresses-topology.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Assigning IP addresses based on topology
+---
+
+## About IP address assignment
+
+{{site.prodname}} can be configured to use specific IP pools for different
+topological areas. For example, you may want workloads in a particular rack,
+zone, or region to receive addresses from the same IP pool. This may be
+desirable either to reduce the number of routes required in the network or to
+meet requirements imposed by an external firewall device or policy.
+
+There are three approaches to configuring IP address assignment behavior which
+the [IPAM section of the cni-plugin configuration reference
+document]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#ipam)
+explains in detail. For the purposes of topology, IP address
+assignment must be per-host (node) which disqualifies using Kubernetes annotations
```

Comment:
Sentence is a little off grammatically but you could just delete "using" and that will fix it. 👍